### PR TITLE
Fix monaco editor not being loaded when switching between css console and sql console

### LINKED ISSUE #5452: Monaco Editor Not Loaded If Open "Custom CSS" page then Sql Console page (or similar)
## Description of bug
In DNN 9.11 the monaco editor isn't loaded if you open the settings => sql console and then the settings => custom css page. This is because they both use the same html id attribute (id="monaco-editor"). I notice that the configuration manager config files & merge scripts panes have their own IDs. I'd recommend the same be done for the 'custom css' page.

## Steps to reproduce
List the precise steps to reproduce the bug:
1. Go to Settings => Sql Console
2. Close it
3. Go to settings => Custom CSS
4. => code editor isn't loaded.

## Current behavior
Code editor isn't loaded when open sql console and css console in same session

## Expected behavior
Code editor is loaded when open sql console and css console in same session

## Screenshots
If applicable, provide screenshots to help explain the bug.
image

## Affected version
* [ ] 10.00.00 alpha build
* [X] 09.11.00 release candidate
* [ ] 09.10.02 release candidate
* [ ] 09.10.01 latest supported release

I've tested 9.11, it may be present in earlier versions too. It'll be happening since the switch to the monaco editor. Looks like september 2020. https://github.com/dnnsoftware/Dnn.Platform/search?q=id%3D%22monaco-editor

## Affected browser
All

### PR DESCRIPTION
**Pull request by mwigley-trilogy**
Fixes #5452

## Summary

Fix monaco editor not being loaded when switching between css console and sql console - see #5452

### CODE REVIEW

File: Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SqlConsole/SqlConsole.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h3 data-bind="html: resx.Title" class="title-small"></h3>
             </div>
         </div>
         <div class="form-item">
-            <div id="monaco-editor"></div>
+            <div id="monaco-editor-css"></div>
```
Comment (bdukes) [inline Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SqlConsole/SqlConsole.html:26]:
```suggestion
            <div id="monaco-editor-sql"></div>
```

File: Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SqlConsole/scripts/SqlConsole.js [hunk 2]
```diff
@@ -780,7 +780,7 @@ define(['jquery',
         theme = e.matches ? "vs-dark" : "vs-light";
       });
 
-      var monacoEditor = sqlConsole.create(document.getElementById("monaco-editor"), {
+      var monacoEditor = sqlConsole.create(document.getElementById("monaco-editor-css"), {
```
Comment (bdukes) [inline Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SqlConsole/scripts/SqlConsole.js:783]:
```suggestion
      var monacoEditor = sqlConsole.create(document.getElementById("monaco-editor-sql"), {

```

**Review by bdukes — APPROVED** [review]
thanks!

**Review by david-poindexter — CHANGES_REQUESTED** [review]
Great catch @mwigley-trilogy - I'll approve when @bdukes change suggestions are committed.

**Comment by mwigley-trilogy** [comment]
Ah whoops, I meant to change the css files. that's what I get for making changes in a hurry! anyway changing the sql one instead is fine. It'll just mean more of the internal ignite integration tests are broken hah. But at least this change will fix some of them too!

**Review by david-poindexter — APPROVED** [review]

**Pull request merged**